modules/ROOT/pages/cache.adoc

```diff
@@ -0,0 +1,49 @@
+= Optionally Configure the Platform Cache
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+You can improve the performance of pages and components in your API community by using the https://help.salesforce.com/articleView?id=data_platform_cache.htm&type=5[Platform Cache, role=external, window=blank].
+
+The platform cache has two parts: a session cache that holds data specific to one user and their session, and an organization cache that holds data that will be accessed by multiple users in the organization.
+
+If you wish to change the default configuration for the caches:
+
+. Log in to your Salesforce organization.
+. In *Setup*, click *Custom Metadata Types*.
+. Next to the label *ACM Platform Cache Metadata*, click *Manage Records*.
+. Next to the label *ACM Configuration*, click *Edit*.
+. Use the *Session Cache Enabled* and *Organization Cache Enabled* checkboxes to enable or disable the corresponding cache.
+. Set the number of seconds that cached data remains in the cache for Applications, API versions, documentation content, instances, and queries with the *Applications*, *API Versions Lifetime*, *API Documentation Lifetime*, *API Instances Lifetime*, and *API Query Lifetime* fields.
```

Comment:
```suggestion
. Set the number of seconds that cached data remains in the cache for applications, API versions, documentation content, instances, and queries by completing the associated fields.
```
It looks like this step is actually the parent to two information points/optional actions? If so, end this with a colon and format the next two steps as unordered list items.

Reply:
agree